eml/services/StorageService.js

```diff
@@ -151,20 +151,147 @@ export const refreshSectionList = async (course_id) => {
           total: section.totalPoints,
         });
       }
-      // Save new courseList for this key and return it.
-      await AsyncStorage.setItem(SECTION_LIST, JSON.stringify(newSectionList));
+    } else {
+      throw new Error('No data to be read in DB or local storage');
```

Comment:
Is this error message (along with the others) meant to be showed to the user? If yes, then it should be in portoguese

Reply:
No, the user are not going to see these